src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+@import "../utils/_variables.scss";
+
+.calendar {
+  display: flex;
+  padding: 20px;
+  gap: $gap;
+  flex-wrap: wrap;
+  width: calc($cell_size * 7 + $gap * 6 + $padding * 3);
+
+  &__cell {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    position: relative;
+    width: 100px;
+    height: 100px;
+    border: 1px solid black;
+    background-color: #eee;
+  }
+
+  &__cell::before {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+  }
+
+  &__cell:hover {
+    cursor: pointer;
+    transform: translateY(-20px);
+    background-color: pink;
+    transition: transform 0.5s;
```

Comment:
move it from the pseudo-class to the class for transition to work when cursor leaves the cell as well